blockchain.go

```diff
@@ -24,19 +24,22 @@ type Signidice struct {
 	Signature string `json:"sign"`
 }
 
-
-func GetSigndiceTransaction(api *eos.API, contract, casinoAccount string,requestID uint64, signature string) (*eos.SignedTransaction, *eos.PackedTransaction, error) {
+func GetSigndiceTransaction(
+	api *eos.API,
+	contract, casinoAccount string,
+	requestID uint64, signature string,
+) (*eos.SignedTransaction, *eos.PackedTransaction, error) {
```

Comment:
why we return signed trx? I see that it used only for debug, so mb move debug output inside of this function and don't return signed trx?